xml/cap_admin_credhub.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter version="5.0" xml:id="cha.cap.credhub"
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+ <info>
+  <title>Integrating Credhub with &productname;</title>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker></dm:bugtracker>
+   <dm:translation>yes</dm:translation>
+  </dm:docmanager>
+ </info>
+
+ <sect1 xml:id="sec.cap.credhub">
+     <title>Installing the Credhub Client</title>
+ <para>
+     &productname; supports Credhub integration. You should already have
+     a working Credhub instance, a Credhub service on your cluster, then apply 
```

Comment:
Need to include how to enable Credhub, e.g. `sizing.credhub_user.count=1` rather than listing as prerequisites.

Reply:
Where does `sizing.credhub_user.count=1` go?

Reply:
It's a helm value so it can go into the values.yaml file (or during `helm install/upgrade --set <blah>")

https://github.com/SUSE/kubernetes-charts-suse-com/blob/master/stable/cf/values.yaml#L1398